src/color/p5.Color.js

```diff
@@ -47,7 +47,7 @@ p5.Color = function(renderer, vals) {
   this.levels = this._array.map(function(level) {
     return Math.round(level * 255);
   });
-
+  this.name = 'p5.Color';   // for friendly debugger system
```

Comment:
@lmccart 
Because the way we declared our classes, typical ways to check class name would not work ([ref1](https://stackoverflow.com/questions/1249531/how-to-get-a-javascript-objects-class), [ref2](https://stackoverflow.com/questions/12944719/how-does-devtools-determine-an-objects-constructors-name)). 
I tried to resolve the issue by adding `name` property into p5 classes.

Currently affected classes are: 
`color/p5.Color`
`core/p5.Element`
`core/p5.Graphics`
`core/p5.Renderer`
`core/p5.Renderer2D`
`image/p5.Image`
`io/p5.Table`
`io/p5.TableRow`
`io/p5.XML`
`math/p5.Vector`
`typography/p5.Font`
`webgl/p5.Geometry`
`webgl/p5.Matrix`
`webgl/p5.RendererGL`